Routines/Camera.py

```diff
@@ -0,0 +1,24 @@
+import picamera
+import time
+import os
+import threading
+data_files = 0
+camera = picamera.PiCamera()
+alert = 0
+lock = threading.Condition()
+class Camera(threading.Thread()):
+    #inittializing thread
+    def __init__(self, thread_name, thread_ID):
+        threading.Thread.__init__(self)
+        camera.vflip = True
+        data_files = 0
+        for files in os.walk("Data"):
+            data_files += 1
+    #run thread and record
+    def run(self):
+        global alert
+        footage_name = "Data/Footage" + (data_files+1)
+        camera.start_recording(footage_name)
```

Comment:
Using an alert flag and threading might be overcomplicating things. If camera.start_recording() isn't a blocking function then there is no need to thread. Just give the ability to start and stop the recording separately. We can handle the rest of the logic in main.py